src/CostManagement/docs/Get-AzCostManagementExportExecutionHistory.md

```diff
@@ -0,0 +1,147 @@
+---
+external help file:
+Module Name: Az.Cost
+online version: https://docs.microsoft.com/en-us/powershell/module/az.cost/get-azcostmanagementexportexecutionhistory
+schema: 2.0.0
+---
+
+# Get-AzCostManagementExportExecutionHistory
+
+## SYNOPSIS
+The operation to get the execution history of an export for the defined scope by export name.
+
+## SYNTAX
+
+### Get (Default)
+```
+Get-AzCostManagementExportExecutionHistory -ExportName <String> -Scope <String> [-DefaultProfile <PSObject>]
+ [<CommonParameters>]
+```
+
+### GetViaIdentity
+```
+Get-AzCostManagementExportExecutionHistory -InputObject <ICostIdentity> [-DefaultProfile <PSObject>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The operation to get the execution history of an export for the defined scope by export name.
+
+## EXAMPLES
+
+### Example 1: Get execution history of cost management export
+```powershell
+PS C:\> Get-AzCostManagementExportExecutionHistory -Scope "subscriptions/9e223dbe-3399-4e19-88eb-0975f02ac87f" -ExportName "ps-customcolum-t"
+
+Name Type
+---- ----
+```
+
+This command gets execution history of cost management export.
+
+### Example 2: Get execution history of cost management export by object
+```powershell
+PS C:\> $export = Get-AzCostManagementExport -Scope "subscriptions/9e223dbe-3399-4e19-88eb-0975f02ac87f" -ExportName "ps-customcolum-t"
+PS C:\> Get-AzCostManagementExportExecutionHistory -InputObject $export
+
+Name Type
+---- ----
```

Comment:
I think it's better that change this example as `by Piping`:
```powershell
PS C:\> Get-AzCostManagementExport -Scope "subscriptions/9e223dbe-3399-4e19-88eb-0975f02ac87f" -ExportName "ps-customcolum-t" | Get-AzCostManagementExportExecutionHistory
```
or
```powershell
PS C:\> $export = Get-AzCostManagementExport -Scope "subscriptions/9e223dbe-3399-4e19-88eb-0975f02ac87f" -ExportName "ps-customcolum-t"
PS C:\> $export | Get-AzCostManagementExportExecutionHistory
```
And please update this exmaple result after pulling the code.

Reply:
Done